src/components/Labels.tsx

```diff
@@ -13,15 +13,15 @@ export const Labels = ({ personData }: LabelColumnProps) => {
   const showSkjermingskode = skjermingskode && skjermingskode !== 'INGEN';
 
   return (
-    <div className="flex">
-      {showSkjermingskode && (
+    showSkjermingskode && (
+      <div className="flex">
         <Tooltip content={getReadableSkjermingskode(skjermingskode)}>
```

Comment:
Trenger man `div` elementet rundt her? 😊 
```suggestion
      <Tooltip className="flex" content={getReadableSkjermingskode(skjermingskode)}>
```

Reply:
Nei kanskje ikke 👍🏼